addons/crm/models/crm_lead.py

```diff
@@ -1338,11 +1338,24 @@ def action_show_potential_duplicates(self):
         }
         return action
 
-    def action_snooze(self):
+    # review:
+    # Should this be moved to mail.activity.mixin?
+    # It would allow to use the reschedule field widget in any model inheriting from mail.activity.mixin
+    # (which is any model that can have activities set on it)
+    def action_reschedule_today(self):
```

Comment:
Be more precise, as it is linked to "my next activity"
```suggestion
    def action_reschedule_my_next_today(self):
```